### PR TITLE
Clarify that save_screenshot needs CSS selector

### PR DESCRIPTION
**Pull request by beneills**
Update README.md to clarify the current situation as promised in #794.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 #### Bug fixes ####
 *   Check isContentEditable property instead of contenteditable attribute so contenteditable children can be modified
+*   Clarify in README.md that save_screenshot only takes CSS selectors (Ben Eills) [Issue #794]
```
Comment (twalpole) [inline CHANGELOG.md]: Clarifying the README is not a bug fix - please remove

**Comment by twalpole** [comment]
@beneills Thanks -- could you squash the two commits please and then I'll merge

**Comment by beneills** [comment]
Squashed.

**Comment by twalpole** [comment]
Thanks

**Pull request merged**